pkg/utils/test/matchers/matchers_test.go

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package matchers
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NotContainAny", func() {
```

Comment:
```suggestion
var _ = Describe("ContainAnyOf", func() {
```
Might be a leftover of a former naming?

Reply:
Oh, yes, thank you :)